analyses/copy_number_consensus_call/src/scripts/bad_chromosomal_seg_updated_merged.txt

```diff
@@ -0,0 +1,3693 @@
+chr1	0	535988	telo,,,	length,,,
```

Comment:
Where did this file come from? Is there a script that generated it? Or did it come from a different source? 

If the former, we would like to have that script included in the repository. If the latter, we should indicate where the file came from in a README in the base directory of this analysis.

Reply:
@xiehongbo provided me with the file so I will let him explain it. I took the file he gave to me, sorted and merged any overlapping segments.

Reply:
@jashapiro I also would like to clarify something. From a glance, it seems that **bedtools subtract** can do what the "get_rid_bad_segments.py" is doing. 

Are you suggesting that we replace that script with a simple bedtools subtract line instead? It seem like bedtools subtract would do the same job and probably much faster.

Reply:
It depends. I want this analysis to be merged quickly, so I don't really want to add more for you to do. But if it is a quick substitution and you can verify that your code and bedtools work the same way, then it might be a good idea. 

I would suggest that we get this merged in as is since we know it works, and you can think about updating to bedtools in a future PR if you have time.

I am more concerned for this PR with making sure we have all the steps required to generate the bad_chromosomal_seg file. That would include the origin of the files that @hongboxie gave you, as well as the scripts for liftover, sorting and merging, as required.

Reply:
I see. So I think right now I will stick with this and make future PR to update to Bedtools if time permits.

As for the bed_chromosomal_seg file. How do you want me to document my process of making this file? Do I comment my process in a certain file somewhere?
I didn't generate this file with a scrip. I did so manually but I could write script if you think it is necessary.

Reply:
A script would be very much preferred, even if it is just a simple shell script with a series of steps.

Reply:
Since the purpose of the script is to document my process, does it have to be integrated into the pipeline? Or can it just be a standalone file?

Reply:
It does not have to be integrated into the snakemake file, but we might ultimately want to add it to the shell script, though that doesn't need to be done right now. I can also add it into the testing once we have it documented in a first phase.

Reply:
I have made the changes and pushed them to this PR. The changes were:

1) Implementing `re.split` in the .py code
2) Add in the script to generate the black list.